server/transport/websocket/WebSocketTransport.cpp

```diff
@@ -341,6 +347,39 @@ WebSocketTransport::setCipherList (context_ptr context,
 #endif
 }
 
+void
+WebSocketTransport::setMinTlsVersion (context_ptr context,
+    const std::string &min_tls)
+{
+  if (min_tls.empty()) {
```

Comment:
"If not supplied, Kurento will try and resolve the highest supported version of TLS" 

Will this hold true if you are returning here?

Reply:
Yeah.  Kurento already sets up some flags that tell the SSL context to just use generic TLS.  The code for that is here: [Disable SSLv2 and SSLv3...](https://github.com/tigewilliamsAxon/kurento-media-server/pull/2/files#diff-f568d86648a00130ddf61b107fd253a934ff135bbf32b6b80a5efae37b7d2b7fR271)

The way the SSL context's `set_options` works is that it only ever adds on new options to the context, so those old ones are still set.  If the user didn't supply a value for `min_tls`, then this `min_tls.empty()` call evaluates to `true` and we just exit out without adding further restrictions onto the SSL context.

A similar thing happens if they just supply a `tls1.0` as the minimum version; we already allow any versions of TLS from 1.0, 1.1, or 1.2 so setting the minimum to 1.0 doesn't really do anything.  I just left it in as a valid config setting for completeness.